MapboxNavigation/GenericRouteShield.swift

```diff
@@ -96,7 +96,13 @@ public class GenericRouteShield: StylableView {
      */
     static func criticalHash(dataSource: DataSource) -> String {
         let proxy = GenericRouteShield.appearance()
-        let criticalProperties: [AnyHashable?] = [dataSource.font.pointSize, proxy.backgroundColor, proxy.foregroundColor, proxy.borderWidth, proxy.cornerRadius]
+        var backgroundColor = proxy.backgroundColor
+        var foregroundColor = proxy.foregroundColor
+        if #available(iOS 13.0, *), let currentTraitCollection = UIApplication.shared.keyWindow?.traitCollection {
+            backgroundColor = proxy.backgroundColor?.resolvedColor(with: currentTraitCollection)
```

Comment:
@1ec5 @JThramer  - What's the preferred way for me to conditionally include/compile the use of UIColor.resolvedColor(with:) only for Xcode 11 and above?

Reply:
If we had to, it would involve `performSelector` ickiness, but I think we should take this opportunity to drop Xcode 10 support, since we’re dropping all sorts of other legacy support right about now. That would involve updating the readme and changelog and removing the relevant lines from the CircleCI configuration file. I’d recommend doing that in a separate PR for better visibility.